R/event-loop.R

```diff
@@ -370,14 +385,31 @@ el__io_poll <- function(self, private, timeout) {
       private$tasks[[id]] <- NULL
       ## TODO: this should be async
       p$data$process$wait(1000)
-      p$data$process$kill()
+      encoding <- p$data$encoding
+
+      stdout <- switch(
+        px_file_type(p$data$stdout),
+        conn = p$data$buffers$stdout$read(),
```

Comment:
I think we need to mark the encoding on this.

Reply:
Done in https://github.com/gaborcsardi/async/pull/71/commits/de9822f9f9231c9582f4e8f1daa32c4ffb7413f5